packages/reporter/src/runnables/runnables.scss

```diff
@@ -1,16 +1,22 @@
+@use '@cypress/design-system/src/index.scss' as *;
+
+@mixin makeRunnableBorder($color) {
+  border-left: 2px solid $color;
+}
+
 .fa { &:not(.fa-spin) { animation: none; } }
 
 .reporter {
   min-height: 0; // needed for firefox or else scrolling gets funky
 
   .container {
-    box-shadow: 0 1px 2px #aaa;
+    border-bottom: 1px solid $metal-10;
```

Comment:
This will impact more than component testing.

Is it this really necessary to have this change in this PR?

Reply:
No, let's remove it

Reply:
done